app/models.py

```diff
@@ -187,3 +187,30 @@ def is_admin(self):
 @login_manager.user_loader
 def load_user(user_id):
     return User.query.get(int(user_id))
+
+
+class Location(db.Model):
+    __tablename__ = 'locations'
+    id = db.Column(db.Integer, primary_key=True)
+    latitude = db.Column(db.String(50))
+    longitude = db.Column(db.String(50))
+    original_text = db.Column(db.Text)
```

Comment:
what's the `original_text` field for? unclear - maybe rename variable or add a line comment?